ajax/libs/monaco-editor/package.json

```diff
@@ -21,8 +21,7 @@
     {
       "basePath": "",
       "files": [
-        "min/**/*",
-        "min-maps/**/*"
+        "(min|min-maps)/**/*"
```

Comment:
Here's `+(min|min-maps)/**/*`:

```
→ node -p 'require("minimatch")("min/file.js", "+(min|min-maps)/**/*")'
true
→ node -p 'require("minimatch")("minmin/file.js", "+(min|min-maps)/**/*")'
true
→ node -p 'require("minimatch")("minmin-maps/file.js", "+(min|min-maps)/**/*")'
true
```

Here's `{min,min-maps}/**/*`:

```
→ node -p 'require("minimatch")("min/file.js", "{min,min-maps}/**/*")'
true
→ node -p 'require("minimatch")("minmin/file.js", "{min,min-maps}/**/*")'
false
→ node -p 'require("minimatch")("minmin-maps/file.js", "{min,min-maps}/**/*")'
false
```

I've changed it to `{min,min-maps}/**/*`.